src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/adjustments/AdjudicationsService.kt

```diff
@@ -40,6 +53,55 @@ class AdjudicationsService(
   override suspend fun retryCreateMapping(message: String) {}
 }
 
+internal fun ReportedAdjudicationResponseV2.toNomisAdjudication() = CreateAdjudicationRequest(
+  adjudicationNumber = reportedAdjudication.adjudicationNumber + 1_000_000, // So we can test basic skeleton in T3 a cheeky hack to ensure number is way out of range of the sequence
+  incident = IncidentToCreate(
+    reportingStaffUsername = reportedAdjudication.createdByUserId,
+    incidentDate = reportedAdjudication.incidentDetails.dateTimeOfIncident.toLocalDate(),
+    incidentTime = reportedAdjudication.incidentDetails.dateTimeOfIncident.toLocalTime().toString(),
```

Comment:
Seems a bit weird to have one as `LocalDate` but the other as `String` - or is there a good reason?

Reply:
It was how the OpenApi dto was created. I assumed it was a known bug. Did you not see this in activity?

Reply:
Yes same! 😮 I might have a little look into it.